ricecooker/commands.py

```diff
@@ -89,6 +98,29 @@ def uploadchannel(chef, command='uploadchannel', update=False, thumbnails=False,
     if hasattr(chef, 'download_content'):
         chef.download_content()
 
+    # TODO load csv if exists
+    if os.path.exists(CSV_DIR):
```

Comment:
I'd suggest making a `load_metadata_file_from_csv` function as a corollary  to `save_metadata_file_to_csv` and putting it in the chef instead of having the code here.

Reply:
made the function